frontend/README.md

```diff
@@ -40,3 +40,63 @@ yarn clean # clear previous build artifacts
 2. Fill in the `.env` file with your Infura ID. You only need the Portis and Fortmatic API keys if you plan on using those wallets
 3. Install dependencies with `yarn`
 4. Build for development with `yarn run dev`
+
+## Internationalization
+
+The app is currently available in English and Simplified Chinese.
+
+Basic usauge is as follows:
+
+1. In the corresponding /i18n/locales/`<locale>`.json file add a new key and corresponding text or translation like so:
+`"key-name" : "Sample text"`
+2. Use the following templates to embed the message on the frontend:
+- Inside templates: `{{ $t('key-name') }}`
+- Inside attributes: `:label="$t('key-name')"`
+- Inside scripts, create an instance inside a function using `const vm = getCurrentInstance()!;` then use `vm.$i18n.t('AccountReceiveTable.date-received')`
+
+While embedding longer texts with styles and links, there are a few options:
+
+1. For texts with html tags and styles:
```

Comment:
By "For texts" do you mean text in the template section of a vue component? If so let's just rephrase to make that more clear

Reply:
See below https://github.com/ScopeLift/umbra-protocol/pull/331#discussion_r875145605